src/controllers/user.controller.js

```diff
@@ -96,7 +105,18 @@ class UserController {
           });
 
           if (user.status === 'unverified') {
-            // TODO : SEND EMAIL TO USER
+            const { protocol } = req;
+
+            mailer({
+              name: username,
+              receiver: email,
+              subject: 'Welcome to ErrorSwag',
+              templateName: 'confirm_account',
+              confirm_account_url: `${protocol}://${req.get(
+                'host'
+              )}/api/v1/auth/verify/${token}`
+            });
+
```

Comment:
Nice here 👍, however as I have pointed out earlier on the signup section. The image below shows the response I got upon login:
![image](https://user-images.githubusercontent.com/31288326/61188205-9c957080-a673-11e9-9a56-c472bf323eeb.png)

As a user, I am expected to get an email notification but I didn't get. Is there something I am expected to do so I can get the email notification?